saleor/graphql/shop/mutations/close_circuit_breaker.py

```diff
@@ -0,0 +1,56 @@
+import graphene
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.utils.module_loading import import_string
+
+from ....core.error_codes import ShopErrorCode
+from ....graphql.app.types import App
+from ....permission.enums import SitePermissions
+from ...core import ResolveInfo
+from ...core.doc_category import DOC_CATEGORY_SHOP
+from ...core.mutations import BaseMutation
+from ...core.types import ShopError
+
+# TODO: make breaker_board instance a singleton
```

Comment:
There is nothing wrong with `breaker_board` not being singleton of `BreakerBoard` assuming it's not being constructed before every request. Currently it doesn't hold any data nor logic that'd require it.

Reply:
@wcislo-saleor I was wondering about singleton because if not, then we have issues with InMemoryStorage. It's not possible to test breaker reset mutation if these have different instances of in-mem storage.